tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/README.md

```diff
@@ -0,0 +1,36 @@
+### Glyph Type Sorting
```

Comment:
Ok, I will remove it from the sub-directory for the time being and find a way to incorporate it into the parent directory's readme. Thanks!